node/src/test_utils/recorder.rs

```diff
@@ -408,6 +413,10 @@ pub fn make_accountant_subs_from_recorder(addr: &Addr<Recorder>) -> AccountantSu
         report_routing_service_provided: recipient!(addr, ReportRoutingServiceProvidedMessage),
         report_exit_service_provided: recipient!(addr, ReportExitServiceProvidedMessage),
         report_services_consumed: recipient!(addr, ReportServicesConsumedMessage),
+        report_our_balances_and_qualified_payables: recipient!(
```

Comment:
This looks like a formatting error. Don't forget to run the script :)

Reply:
It's a valid formatting. :man_shrugging: 

The formater forces this layout. I couldn't think of a better one which wouldn't be knocked out by the formatter so I left it untouched.